include/z64save.h

```diff
@@ -24,6 +25,56 @@ typedef enum RespawnMode {
 
 #define SAVE_BUFFER_SIZE 0x4000
 
+#define SECONDS_TO_TIMER(seconds) ((seconds) * 100)
```

Comment:
Hmm, I think you could combine `SECONDS_TO_TIMER_PRECISE` and `SECONDS_TO_TIMER` by just using `SECONDS_TO_TIMER_PRECISE` with centiseconds equal to 0. Thoughts?

Reply:
You can definitely combine them into one, the only reason I made two is that nearly all timers are just in seconds, and having units of centiSeconds doesn't seem intuitive, so I'd figure it'd be easier to read if there was a simple one and a special case one if you needed a more precise time. So I like having two, but will change it to one if others think it's better that way

Reply:
That makes sense. I don't have too strong an opinion either way, and mainly curious as to your thoughts of splitting up.